tests/acceptance/features/cliCommands/uploadSessions.feature

```diff
@@ -190,3 +190,37 @@ Feature: List upload sessions via CLI command
       | large.zip |
     And the CLI response should not contain these entries:
       | file.txt |
+
+
+  Scenario: resume a specific failed upload session
```

Comment:
the scenario title will be confusing since we have similar scenarios above
```suggestion
  Scenario: resume a specific upload session using postprocessing command
```